doc/sphinx/source/_templates/empty

```diff
@@ -0,0 +1 @@
+This directory is empty.
```

Comment:
Yeah, my thoughts exactly, especially since it's likely to become out of date.

Ah, I didn't realize they were auto-generated.